### PR TITLE
upgrade auto-build

### PR DESCRIPTION
**Pull request by Stevent-fei**

### CODE REVIEW

File: .github/workflows/auto-build.yml [hunk 1]
```diff
@@ -19,8 +19,8 @@ jobs:
           echo $version
           arch=`echo "$commentbody"|cut -d" " -f3`
           echo $arch
-          wget http://sealer.oss-cn-beijing.aliyuncs.com/auto-build/rootfs.tar.gz
-          tar -xvf rootfs.tar.gz
+          wget https://sealer.oss-cn-beijing.aliyuncs.com/auto-build/rootfs-07.tar.gz
```
Comment (fanux) [inline .github/workflows/auto-build.yml:22]: Sealer version as an argument too. https://sealer.oss-cn-beijing.aliyuncs.com/sealers/sealer-${sealerversion}-linux-amd64.tar.gz

File: .github/workflows/auto-build.yml [hunk 2]
```diff
@@ -31,16 +31,24 @@ jobs:
           wget http://sealer.oss-cn-beijing.aliyuncs.com/auto-build/sealer.tar.gz
           sudo tar -xvf sealer.tar.gz -C /usr/bin
           sudo sealer login ${{secrets.PRIVATEWAREHOUSE}}
-          sudo sed -i "s/v1.20.14/$version/g" ./rootfs/etc/kubeadm.yml
-          sudo sed -i "s/v1.20.14/$version/g" ./rootfs/Metadata
+          sudo sed -i "s/v1.19.8/$version/g" ./rootfs/etc/kubeadm.yml
```
Comment (fanux) [inline .github/workflows/auto-build.yml:34]: Using `KUBE_VERSION` install v1.19.8

Comment (Stevent-fei) [reply .github/workflows/auto-build.yml:34]: Yes, kubernetes version is obtained in kubedm, kubedm.yaml is a fixed file uploaded to the OSS. Here, we only replace the contents of the file

File: .github/workflows/auto-build.yml (hunk 1 above)
Comment (Stevent-fei) [reply .github/workflows/auto-build.yml]: Substituting sealer-latest achieved the same effect

**Review by fanux — APPROVED** [review]

**Pull request merged**